dashboard/app/models/script.rb

```diff
@@ -620,6 +620,7 @@ def summarize
     summary = {
       id: id,
       name: name,
+      plc: professional_learning_course,
```

Comment:
When you rebase against staging, you can make this professional_course so it includes pd courses as well